test/jdk/java/lang/String/Chars.java

```diff
@@ -45,6 +47,7 @@ public static void main(String[] args) {
             }
             testChars(cc, ccExp);
             testCharsSubrange(cc, ccExp);
+            testIntsSubrange(ccExp);
```

Comment:
Please also add the same call after line 74 to apply the test to the surrogates case.
(As suggested by @rgiulietti in https://github.com/openjdk/jdk/pull/17066)

Reply:
Just pulled the update from that branch.